helm/oauth2-proxy/templates/configmap-htpasswd-file.yaml

```diff
@@ -14,4 +14,9 @@ stringData:
     {{- range $entries := .Values.htpasswdFile.entries }}
     {{ $entries }}
     {{- end -}}
+    {{- range $entry := .Values.htpasswdFile.entriesPlain }}
+      {{- if and $entry.username $entry.password }}
+    {{ htpasswd $entry.username $entry.password }}
```

Comment:
I was going to comment here but instead learned about the `htpasswd` built-in function https://helm.sh/docs/chart_template_guide/function_list/#htpasswd

Reply:
Most important helm functions are provided by Sprig: https://masterminds.github.io/sprig/